tests/test_root_model.py

```diff
@@ -325,17 +326,44 @@ class Model(RootModel[int]):
             model_config = ConfigDict(extra=extra_value)
 
 
-@pytest.mark.xfail(reason='TODO: support applying defaults with `RootModel`')
 def test_root_model_default_value():
     class Model(RootModel):
         root: int = 42
 
     m = Model()
     assert m.root == 42
-    assert m.model_dump == 42
+    assert m.model_dump() == 42
+
+
+def test_root_model_wrong_default_value_without_validate_default():
+    class Model(RootModel):
+        root: int = '42'
+
+    m = Model()
+    assert m.root == '42'
```

Comment:
As it is only one assertion and we don't need `m` after that.

```suggestion
    assert Model().root == '42'
```

up to you to change it or not